### PR TITLE
11879_update_node_version

### PR DESCRIPTION
**Pull request by ozamani9gh**
*Issue #:* /bcgov/entity#11879

*Description of changes:*
update note version from 14.17.5 to 16.14.2

### CODE REVIEW

**Review by severinbeauvais — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "2.1.8",
+  "version": "3.0.0",
```
Comment (severinbeauvais) [inline package.json:3]: No change to `vue-test-utils-helps` spec on line 84?

Comment (ozamani9gh) [reply package.json:3]: i changed the package-lock.json directly to fix the +ssh to +https   as recommended by @pwei1018

Comment (severinbeauvais) [reply package.json:3]: Obsolete comment.

**Review by severinbeauvais — APPROVED** [review]
LGTM but you need to rebase before we can merge this.

**Review by severinbeauvais — APPROVED** [review]
When the merge conflicts are resolved, go ahead and merge this.

Then smoke-test it in Dev!

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**